lib/cursor-position-view.coffee

```diff
@@ -22,7 +26,9 @@ class CursorPositionView extends HTMLElement
 
   updatePosition: ->
     if position = @getActiveTextEditor()?.getCursorBufferPosition()
-      @textContent = "#{position.row + 1}:#{position.column + 1}"
+      @row = position.row + 1
+      @column = position.column + 1
+      @textContent = "#{@row}:#{@column}"
     else
       @textContent = ''
```

Comment:
What happens if you hover over the (admittedly tiny) tile when there is no active text editor? I suspect it'll say `Line whateverTheLastRowWas, Column whateverTheLastColumnWas`. I suspect this else needs to be updated and maybe the tooltip disabled or updated to say something like "no active text editor" in case you accidentally hover over it.